backend/src/controllers/FriendController.ts

```diff
@@ -0,0 +1,202 @@
+import User from '../models/userModel';
+class FriendController {
+  public static async getFriends(req: any, res: any) {
+    const userID: string = req.user.data._id;
+    try {
+      const user = await User.findOne({ _id: userID }).exec();
+      if (!user) {
+        return res.status(404).send({
+          message: `User ${userID} not found`,
+        });
+      }
+      res.send(user.friends);
+    } catch (err) {
+      console.error(err);
+      res.status(500).send({
+        message: `Error while getting User ${userID}`,
+        error: err,
+      });
+    }
+  }
+
+  public static async deleteFriend(req: any, res: any) {
+    const userID: string = req.user.data._id;
+    const friendID: string = req.params.friendID;
+    try {
+      const user = await User.findOne({ _id: userID }).exec();
+      const friend = await User.findOne({ _id: friendID }).exec();
+      if (!user || !friend) {
+        return res.status(404).send({
+          message: 'One of the users is missing!',
+        });
+      }
+      if (!user.friends.includes(friendID)) {
+        return res.status(404).send({
+          message: `User is not friends with ${
+            (friend.firstName, +' ' + friend.lastName)
```

Comment:
**being super nit picky** but maybe run the formatter in this file to avoid formatting issues like this one: `+' ' + friend.lastName`